doc/pprof.md

```diff
@@ -105,10 +105,58 @@ number of values - 1) or the name of the sample value.
 
 Sample values are numeric values associated to a unit. If pprof can recognize
 these units, it will attempt to scale the values to a suitable unit for
-visualization. The `unite=` option will force the use of a specific unit. For
+visualization. The `unit=` option will force the use of a specific unit. For
 example, `sample_index=sec` will force any time values to be reported in
 seconds. pprof recognizes most common time and memory size units.
 
+## Tag Filtering
+
+Samples in a profile may have tags. These tags have a name and a value; this
+value can be either numeric or a string. pprof can select samples from a profile
+based on these tags using the `-tagfocus` and `-tagignore` options.
+
+Generally, these options work as follows:
+* **-tagfocus=_regex_** or **-tagfocus=_range_:** Restrict to samples with tags
+  matched by regexp or in range
+* **-tagignore=_regex_** or **-tagignore=_range_:** Discard samples with tags
+  matched by regexp or in range
+
+When using `-tagfocus=regex` and `-tagignore=regex`, the regex will be compared
+to each value associated with each tag. If one specifies a regex
+like `regex1,regex2`, then only samples with a tag value matching `regex1` and a
+tag value matching `regex2` will be kept.
+
+In addition to being able to filter on tag values, one can specify the name of the
+tag which a certain value must be associated with using the notation
+`-tagfocus=tagName=value1`. Here, the `tagName` must match the tag's name exactly,
+and the value can be either a regex or a range. If one specifies a regex
+like `regex1,regex2`, then samples with a value associated with the given tag name
```

Comment:
> then samples with a value associated with the given tag name matching either `regex1` or matching `regex2` will match

I find this text a bit confusing - it is not super-clear that it's tag value that is matched against the regex, not tag name.